requirements.txt

```diff
@@ -5,3 +5,4 @@ pandas~=0.19; python_version >= '3.5'
 pandas>=0.19,<0.21; python_version == '3.4'
 pandas>=0.19,<=0.23; python_version == '2.7'
 six~=1.10
+joblib>=0.17.0
```

Comment:
Could you please change the numpy line to `numpy>=1.12,<2.0`? It sounds like `MaskedArray` objects in previous versions of numpy were not picklable.

https://github.com/scikit-learn/scikit-learn/blob/0.20.X/sklearn/utils/fixes.py#L241

Reply:
Yup, noticed that but wasn't sure if that mattered for us. Cool, I'll go ahead and do that.